sycl/test/basic_tests/kernel_bundle_has_kernel.cpp

```diff
@@ -0,0 +1,18 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -c -o %t.out
```

Comment:
If you don't plan to launch the binary, then you should use `-fsyntax-only` instead of `-o`: this runs less compiler stages and does not produce any output files, which is better if you only want to check that the code compiles.t

However, I would anyway recommend to add a unit-test, which checks that the new method actually works as expected